intro/numpy/array_object.rst

```diff
@@ -432,12 +432,11 @@ Or, if you have enabled interactive plots with ``%matplotlib``:
 
 .. sourcecode:: pycon
 
-  >>> x = np.linspace(0, 3, 20)
-  >>> y = np.linspace(0, 9, 20)
-  >>> plt.plot(x, y)       # line plot    # doctest: +SKIP
-  [<matplotlib.lines.Line2D object at ...>]
-  >>> plt.plot(x, y, 'o')  # dot plot    # doctest: +SKIP
-  [<matplotlib.lines.Line2D object at ...>]
+  >>> image = np.random.rand(30, 30)
+  >>> plt.imshow(image, cmap=plt.cm.hot)    # doctest: +ELLIPSIS
```

Comment:
Is there a reason to use specifically the hot colormap here ? (I lack the context)

Reply:
I just copied the content of `intro/numpy/examples/plot_basic2dplot.py` as there was a misalignment between the example and the plot. BTW, we don't have a way to include the example code directly - or generating the examples code and plot from the rst?